### PR TITLE
2624/add cow to swap modal

### LINKED ISSUE #2624: Add COW token to favorite token lists in swap modal

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Fixes #2624

- adds COW to common tokens list
- moves COW up the token list

### CODE REVIEW

File: src/custom/components/SearchModal/CurrencySearch/sorting.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    return balanceA.greaterThan(balanceB) ? -1 : balanceA.equalTo(balanceB) ? 0 : 1
+  } else if (balanceA && balanceA.greaterThan('0')) {
+    return -1
+  } else if (balanceB && balanceB.greaterThan('0')) {
+    return 1
+  }
+  return 0
+}
+
+function getTokenComparator(balances: {
```
Comment (W3stside) [inline src/custom/components/SearchModal/CurrencySearch/sorting.ts:18]: didn't this exist somewhere already? or did u just move this and mod it a bit?

**Comment by W3stside** [comment]
nice small PR! thank you!! will review in depth a little later

File: src/custom/components/SearchModal/CurrencySearch/sorting.ts (hunk 1 above)
Comment (nenadV91) [reply src/custom/components/SearchModal/CurrencySearch/sorting.ts:18]: It exists, this is mod

Comment (W3stside) [reply src/custom/components/SearchModal/CurrencySearch/sorting.ts:18]: maybe it's worth making a const list of tokens we'd like to prioritise listed in order of importance

e.g `const PRIORITISED_TOKENS = [COW, GNO]` where lower index = higher prio

**Comment by elena-zh** [comment]
Hey @nenadV91 , great!
The only thing I can mention, that we do not have this token in the list for Rinkeby
image

**Comment by nenadV91** [comment]
@elena-zh I think we should do that in a separate PR since its not related to this issue.

**Pull request closed without merging**